flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointStateRestoreTest.java

```diff
@@ -66,10 +66,19 @@
 	public void testSetState() {
 		try {
 
-			final ChainedStateHandle<StreamStateHandle> serializedState = CheckpointCoordinatorTest.generateChainedStateHandle(new SerializableObject());
-			KeyGroupRange keyGroupRange = KeyGroupRange.of(0,0);
-			List<SerializableObject> testStates = Collections.singletonList(new SerializableObject());
-			final KeyedStateHandle serializedKeyGroupStates = CheckpointCoordinatorTest.generateKeyGroupState(keyGroupRange, testStates);
+			final ChainedStateHandle<StreamStateHandle> serializedState1 = CheckpointCoordinatorTest.generateChainedStateHandle(new SerializableObject());
```

Comment:
Could you write in commit message what are actually fixing? And could you also add some commit title that sums up your changes?

After this commit, will this test start failing and only next commits fix that?